### PR TITLE
sportacus

### PR DESCRIPTION
**Pull request by starch70**
## About the PR
my admin event antag will have stun immunity, health regen, can instantly kill people

## Why / Balance
To be fair, you have to have a very high IQ to understand Rick and Morty. The humor is extremely subtle, and without a solid grasp of theoretical physics most of the jokes will go over a typical viewer's head. There's also Rick's nihilistic outlook, which is deftly woven into his characterisation - his personal philosophy draws heavily fromNarodnaya Volya literature, for instance. The fans understand this stuff; they have the intellectual capacity to truly appreciate the depths of these jokes, to realize that they're not just funny- they say something deep about LIFE. As a consequence people who dislike Rick and Morty truly ARE idiots- of course they wouldn't appreciate, for instance, the humour in Rick's existencial catchphrase "Wubba Lubba Dub Dub," which itself is a cryptic reference to Turgenev's Russian epic Fathers and Sons I'm smirking right now just imagining one of those addlepated simpletons scratching their heads in confusion as Dan Harmon's unfolds itself on their television screens. What fools... how I pity them. 😂 And yes by the way, I DO have a Rick and Morty tattoo. And no, you cannot see it. It's for the ladies' eyes only- And even they have to demonstrate that they're within 5 IQ points of my own (preferably lower) beforehand.

## Media
image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**
:cl:
- add: added sportacus

## Summary by CodeRabbit

- **New Features**
	- Introduced a new entity "Sportacus" with unique mechanics, including high movement speed, powerful melee weapon, and special hunger/thirst systems.
	- Added a visual representation for "Sportacus" with defined animation states for alive and dead conditions.

- **Documentation**
	- Added a metadata file for "Sportacus" textures, detailing animation states and licensing information.

### CODE REVIEW

**Comment by deltanedas** [comment]
he ate his fruit... its over for the valids

**Comment by Aineias1** [comment]
7

**Review by whateverusername0 — APPROVED** [review]
pr of the year, merge this now

File: Resources/Prototypes/_Goobstation/Entities/Mobs/NPCs/miscellaneous.yml [hunk 1]
```diff
@@ -203,3 +203,95 @@
...
+    requiredLegs: 0
+  - type: MeleeWeapon
+    damage:
+      types:
+        Blunt: 200000
+  - type: Butcherable
+    butcheringType: Spike
+    spawned:
+    - id: FoodMeat
+      amount: 30000
```
Comment (Piras314) [inline Resources/Prototypes/_Goobstation/Entities/Mobs/NPCs/miscellaneous.yml:230]: i dare you to butcher this on your pc :godmode: stop trying to crash the server!!

Comment (starch70) [reply Resources/Prototypes/_Goobstation/Entities/Mobs/NPCs/miscellaneous.yml:230]: if sportacus dies somehow the server deserves to crash

**Comment by JoeHammad1844** [comment]
woe

**Comment by Scruq445** [comment]
I forgot to remove the "poisoning" effect from flour

File: Resources/Prototypes/Reagents/Consumable/Food/ingredients.yml [hunk 2]
```diff
@@ -18,6 +18,13 @@
       - !type:AdjustReagent
         reagent: Nutriment
         amount: 0.1
+      - !type:HealthChange
+        conditions:
+        - !type:OrganType
+          type: Athletic
+        damage:
+          groups:
+            Poison: 100000000
```
[the remark this answers is not in the document]
Comment (Scruq445) [reply Resources/Prototypes/Reagents/Consumable/Food/ingredients.yml:27]: I forgot to remove that code from flour

**Comment by Piras314** [comment]
`Error: /Prototypes/Reagents/Consumable/Drink/juice.yml(48,9)  Field "type" not found in "Content.Server.EntityEffects.Effects.MovespeedModifier".
Error: /Prototypes/Reagents/Consumable/Drink/juice.yml(49,9)  Field "time" not found in "Content.Server.EntityEffects.Effects.MovespeedModifier".
Error: /Prototypes/_Goobstation/Chemistry/metabolizer_types.yml(3,9)  No localization message found with id Athletic
2 errors found in 66[7](https://github.com/Goob-Station/Goob-Station/actions/runs/11592447926/job/32274331053?pr=808#step:9:8)01 ms.
Error: Process completed with exit code 255.`

**Review by Aidenkrz — CHANGES_REQUESTED** [review]
remove all the stupid ass apple juice shit and just have the npc and i guess i'll merge this

**Pull request merged**